### PR TITLE
feat(issue): add platform description to bug-report

### PR DESCRIPTION
**Pull request by lhdjply**
Due to different situations occurring in different development environments, the development environment should be specified when reporting bugs.

### Notes
- Update the [Documentation](https://github.com/lvgl/lvgl/tree/master/docs) if needed.
- Add [Examples](https://github.com/lvgl/lvgl/tree/master/examples) if relevant.
- Add [Tests](https://github.com/lvgl/lvgl/blob/master/tests/README.md) if applicable.
- If you added new options to `lv_conf_template.h` run [lv_conf_internal_gen.py](https://github.com/lvgl/lvgl/blob/master/scripts/lv_conf_internal_gen.py) and update [Kconfig](https://github.com/lvgl/lvgl/blob/master/Kconfig).
- Run `scripts/code-format.py` (`astyle v3.4.12` needs to installed by running `cd scripts; ./install_astyle.sh`) and follow the [Code Conventions](https://docs.lvgl.io/master/CODING_STYLE.html).
- Mark the Pull request as [Draft](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/changing-the-stage-of-a-pull-request) while you are working on the first version, and mark is as _Ready_ when it's ready for review.
- When changes were requested, [re-request review](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/requesting-a-pull-request-review) to notify the maintainers.
- Help us to review this Pull Request! Anyone can [approve or request changes](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/approving-a-pull-request-with-required-reviews).

### CODE REVIEW

**Review by FASTSHIFT — DISMISSED** [review]

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 1]
```diff
@@ -17,6 +17,14 @@ body:
...
     validations:
       required: true
+  - type: input
+    id: development_environment
+    attributes:
+      label: Development environment
+      description: Tell us what development environment you use
+      placeholder: e.g. Visual Studio
+    validations:
+      required: true
```
Comment (kisvegabor) [inline .github/ISSUE_TEMPLATE/bug-report.yml]:
```suggestion
    id: platform
    attributes:
      label: Platform
      description: Tell us platform and environment you use
      placeholder: e.g. Visual Studio simulator, or STM32F769 Discovery with Zephyr, or Eclipse with SDL on Linux 
    validations:
      required: true
```

**Review by kisvegabor — COMMENTED** [review]
We should cover embedded platforms too, so I think "platform" is more generic word. What do you think?

File: .github/ISSUE_TEMPLATE/bug-report.yml (hunk 1 above)
Comment (lhdjply) [reply .github/ISSUE_TEMPLATE/bug-report.yml]: Yes, the word platform does make more sense.

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 2]
```diff
@@ -18,11 +18,11 @@ body:
...
       required: true
   - type: input
-    id: development_environment
+    id: platform
     attributes:
-      label: Development environment
-      description: Tell us what development environment you use
-      placeholder: e.g. Visual Studio
+      label: Platform
+      description: Tell us platform and environment you use
```
Comment (kisvegabor) [inline .github/ISSUE_TEMPLATE/bug-report.yml]:
```suggestion
      description: Tell us about the platform and environment you use
```

**Review by kisvegabor — APPROVED** [review]
Thank you

**Review by FASTSHIFT — APPROVED** [review]

**Comment by kisvegabor** [comment]
Let's change it to a `textarea`. Hopefully it will show people that it's as important as the actual bug report, and motivate them to fill it. :smile:

**Comment by lhdjply** [comment]
> Let's change it to a `textarea`. Hopefully it will show people that it's as important as the actual bug report, and motivate them to fill it. 😄

Change it  in #7316

**Pull request merged**